src/Sulu/Bundle/ContentBundle/Admin/ContentAdmin.php

```diff
@@ -104,6 +104,27 @@ public function __construct(
         $this->setNavigation(new Navigation($rootNavigationItem));
     }
 
+    public function getNavigationV2(): Navigation
+    {
+        $rootNavigationItem = Admin::getNavigationItemRoot();
```

Comment:
This entire static method would be obsolete if we change the admin as described above and return an array of Navigation items.